rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -37,6 +37,7 @@ static bool initialized = false;
 static char *sgx_dev_path;
 static bool no_sgx_flc = false;
 static bool fork_test = false;
+static bool enclave_debug = false;
```

Comment:
Everything looks good except the default value of enclave_debug. There are non-trivial machines running sgx1 without FLC so you still need to set the default value of enclave_debug to true.